src/pages/basic_data/institution_admin/AddEmployeePage.tsx

```diff
@@ -68,7 +68,13 @@ export const AddEmployeePage = () => {
       });
       const createPersonResponse = await createCristinPerson(cristinPerson);
       if (isErrorStatus(createPersonResponse.status)) {
-        dispatch(setNotification({ message: t('feedback.error.create_user'), variant: 'error' }));
+        dispatch(
+          setNotification({
+            message: t('feedback.error.create_user'),
+            variant: 'error',
+            detail: (createPersonResponse.data as any).detail,
```

Comment:
Uheldig å måtte bruke `any` her, men har ikke funnet en enkel måte å håndtere det på. Må nok fikse return-type på `apiRequest` til noe som `<AxiosResponse<Promise<T | ErrorResponse>>`, men det har ganske store ringvirkninger, så det blir litt out of scope for denne PR'en.

Reply:
Suger litt ja, men ser ingen god løsning jeg heller nei :/

Reply:
Kunne ev. brukt `'detail' in createPersonResponse.data` da